src/ImmutableX.ts

```diff
@@ -592,19 +592,17 @@ export class ImmutableX {
 
   /**
    * Completes a Withdrawal
-   * @param ethSigner - the L1 signer
-   * @param starkPublicKey - the Signer address
+   * @param walletConnection - the pair of L1/L2 signers
```

Comment:
Parameter change necessary because `completeWithdrawal` needs stark signer in order to do `registerEthAddress` in StarkEx V4

https://github.com/immutable/imx-core-sdk/pull/423/files#diff-1ec0606bbd2c1c2767dfceb18d6dcade2beeedc6479166251fe7b86cca8d072b